### ISSUE TITLE
[Feature Request] Different screen brightness on battery and power supply

### ISSUE THREAD

**Issue by AVE-RVV** [linuxmint/cinnamon#9765]
Dear all,

I think it would be very nice and useful to adjust different screen brightness on battery and power supply.

for example:
![Screenshot from 2020-12-11 01-27-16](https://user-images.githubusercontent.com/75805766/101861476-bad20980-3ba2-11eb-851d-14f97c901105.png)

* cinnamon 4.6.7
 * Distribution - Linux Mint 20 Ulyana
 * Intel
 * 64 bit

**Comment by Perkolator** [comment]
Yes, laptop users would be happy with this feature. The "on battery, dim screen when inactive" feature is a completely different feature than what is requested here. It would be great to have the brightness lowered instantly when entering battery mode, whether the user is actively using the laptop or not. I was kind of surprised to see that this is not currently available in Mint/Cinnamon.
